calculator.js

```diff
@@ -1,5 +1,69 @@
+// define four example constants to simulate user input
 const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+  num1: 8,
+  num2: 8.2,
+  operation: ['add', '+']
 }
+
+const exampleSubtractionInput = {
+  num1: 28,
+  num2: -80,
+  operation: ['subtract', '-']
+}
+
+const exampleMultiplicationInput = {
+  num1: 5,
+  num2: 9,
+  operation: ['multiply', '*']
+}
+
+const exampleDivisionInput = {
+  num1: 100,
+  num2: 4,
+  operation: ['divide', '/']
+}
+
+// helper methods for four operations
+const add = (num1, num2) => num1 + num2;
+const subtract = (num1, num2) => num1 - num2;
+const multiply = (num1, num2) => num1 * num2;
+const divide = (num1, num2) => num1 / num2;
+
+// helper method to validate input numbers
+const isNumValid = function(input) {
+  return (typeof(input) === 'number' ? true : false);
+}
+
+// main calculator logic
+const calculate = function(input) {
+  // validate two input nums
+  if (!isNumValid(input.num1) || !isNumValid(input.num2)) {
+    return "Oops! Input must be a number.";
+  } else {
+    let result = null;
+    // only word and symbol allowed for each operation
+    if (input['operation'].includes('add') || input['operation'].includes('+')) {
+      result = add(input['num1'], input['num2']);
+    } else if (input['operation'].includes('subtract') || input['operation'].includes('-')) {
+      result = subtract(input['num1'], input['num2']);
+    } else if (input['operation'].includes('multiply') || input['operation'].includes('*')) {
+        result = multiply(input['num1'], input['num2']);
+    } else if (input['operation'].includes('divide') || input['operation'].includes('/')) {
```

Comment:
Using `.includes` here to determine the operation works with the way you've built your inputs above, but I'm not sure it would make sense in the context of a bigger program.

Saying `input['operation'].includes` implies that `input['operation']` is an array. But if we imagine taking input from the user, they're probably only going to provide one operation, not a whole list of them.

Instead, I would probably have `input['operation']` be a single value like `+` or `subtract`, and check for equality in these conditionals.